keyboards/fourier/rev1/config.h

```diff
@@ -37,6 +34,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 // wiring of each half
 #define MATRIX_ROW_PINS { F4, D7, E6, B4 }
 #define MATRIX_COL_PINS { F5, F6, F7, B1, B3, B2, B6 }
+#define SPLIT_HAND_PIN D2
 
```

Comment:
Could you add this too:
```suggestion

/* serial.c configuration for split keyboard */
#define SOFT_SERIAL_PIN D0

```

Reply:
The last comment on #4419 was about providing defaults for this value, given that other parts of the code base as well as split functionality provide defaults, should this really be required?

I'm happy to add if required, but it kinda feels dirty given how everything else works out of the box.

Reply:
Cool, let me fire up my environment and make the required changes.

Reply:
Done.